### PR TITLE
init Laravel project 

### PR DESCRIPTION
**Pull request by brunobuddy**
Create Customer model
Add entrypoint GET /api/customers/$id?token=$token

We match Customer Id and Token to validate before sending information to interface.

### CODE REVIEW

File: database/migrations/2016_11_26_152122_create_customers_table.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('customers', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('source_id');
```
Comment (n1c01a5) [inline database/migrations/2016_11_26_152122_create_customers_table.php]: necessary ?

Comment (n1c01a5) [inline database/migrations/2016_11_26_152122_create_customers_table.php]: necessary ?

I think we just need `first_name`, `last_name`, `email`, `phone` and `amount`

File: database/migrations/2016_11_26_152122_create_customers_table.php [hunk 2]
```diff
@@ -20,6 +20,7 @@ public function up()
             $table->string('flight_id');
             $table->string('email');
             $table->string('phone_number');
```
Comment (n1c01a5) [inline database/migrations/2016_11_26_152122_create_customers_table.php]: string ? not integer ?

File: database/migrations/2016_11_26_152122_create_customers_table.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('customers', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('source_id');
+            $table->string('first_name');
+            $table->string('last_name');
+            $table->string('flight_id');
```
Comment (n1c01a5) [inline database/migrations/2016_11_26_152122_create_customers_table.php]: integer format ?

File: app/Events/Event.php [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace App\Events;
+
+abstract class Event
+{
+    //
```
Comment (brunobuddy) [inline app/Events/Event.php]: default

File: database/migrations/2016_11_26_152122_create_customers_table.php (hunk 1 above)
Comment (brunobuddy) [inline database/migrations/2016_11_26_152122_create_customers_table.php]: done

File: database/migrations/2016_11_26_152122_create_customers_table.php (hunk 2 above)
Comment (brunobuddy) [inline database/migrations/2016_11_26_152122_create_customers_table.php]: done

File: database/migrations/2016_11_26_152122_create_customers_table.php [hunk 5]
```diff
@@ -20,6 +20,7 @@ public function up()
             $table->string('flight_id');
```
Comment (brunobuddy) [inline database/migrations/2016_11_26_152122_create_customers_table.php]: done

File: database/migrations/2016_11_26_152122_create_customers_table.php (hunk 3 above)
Comment (brunobuddy) [inline database/migrations/2016_11_26_152122_create_customers_table.php]: done

**Pull request merged**